tox.ini

```diff
@@ -45,6 +45,8 @@ deps =
     {[matplotlib]deps}
     kaleido
     plotly
+    kaleido ; platform_system != 'Windows'
+    kaleido==0.1.0.post1 ; platform_system == 'Windows'
```

Comment:
Hm if this doesn't work try `kaleido==0.1.0.post1 ; sys_platform == 'windows'`

the example they provide is `pywin32 >=1.0 ; sys_platform == 'win32'` so not exactly the same but maybe it understands both idk

Reply:
I thought the syntax would be the same as pyproject.toml but silly me to expect uniformity in the python packaging world...

Reply:
well looks like it worked actuallt!